sdcm/cluster.py

```diff
@@ -1719,6 +1719,11 @@ def download_scylla_manager_repo(self, scylla_repo: str) -> None:
         else:
             repo_path = '/etc/apt/sources.list.d/scylla-manager.list'
         self.remoter.sudo(f"curl -o {repo_path} -L {scylla_repo}")
+
+        # Prevent issue https://github.com/scylladb/scylla/issues/9683
+        if self.distro.is_oel8:
+            self.remoter.sudo(f"chmod 644 {repo_path}")
```

Comment:
if the issue is being fixed in Scylla, do we need to set this permission in SCT too?

Reply:
The issue explains about 2 problems. One problem was fixed in Scylla. Second, permission problem of scylla-manager.repo, on us, because we are responsible for manager installation. So we should fix it